.travis.yml

```diff
@@ -39,7 +44,3 @@ script: ./travis.sh
 notifications:
   email:
     - cukes-devs@googlegroups.com
-
-branches:
-  only:
-    - master
```

Comment:
Ah yes, I needed that to be able to test in my own fork. I'll split it off to a separate PR